src/sfctl/params.py

```diff
@@ -214,3 +214,7 @@ def custom_arguments(self, _):  # pylint: disable=too-many-statements
         arg_context.argument('secret_resource_name')
         arg_context.argument('secret_value_resource_name')
         arg_context.argument('show_value')
+
+    with ArgumentsContext(self, 'mesh deployment create') as arg_context:
+        arg_context.argument('input_yaml_file_paths')
```

Comment:
This seems like an oversight on my part in the last review. There should be no need to list the arguments here. Same with 'mesh secretvalue show' above. If the input type is a string, then there's no need to list the parameters in here. Can you remove this and the 'mesh secretvalue show' above? Thanks!